scripts/processing/Makefile

```diff
@@ -69,5 +74,6 @@ profile:
 	mprof plot -o group_average_source
 
 all:
-	$(MAKE) fetch-recon
+	$(MAKE) fetch
+	$(MAKE) recon
```

Comment:
need to update the help commands too, but I can do it. Just let me know about the `make clean`